packages/components/src/components/CollapsibleBox/CollapsibleBox.tsx

```diff
@@ -121,25 +111,21 @@ const CollapsibleBox: React.FC<CollapsibleBoxProps> & CollapsibleBoxSubcomponent
     subHeading,
     iconLabel,
     children,
-    noContentPadding,
     variant = 'small',
-    opened = false,
+    isOpened = false,
     onCollapse,
-    headingButton,
     ...rest
 }: CollapsibleBoxProps) => {
-    const [collapsed, setCollapsed] = useState(!opened);
-    const [animatedIcon, setAnimatedIcon] = useState(false);
+    const [isCollapsed, setIsCollapsed] = useState(!isOpened);
 
     useEffect(() => {
-        setCollapsed(!opened);
-    }, [opened]);
+        setIsCollapsed(!isOpened);
+    }, [isOpened]);
```

Comment:
Does this even do anything? :)

Reply:
sets the local `isCollapsed` state when the `isOpen` prop changes so that it can be externally controlled.